vendor/github.com/turbonomic/turbo-api/pkg/client/request.go

```diff
@@ -183,10 +184,10 @@ func (r *Request) request(fn func(*http.Response)) error {
 			req.Header.Set(key, value)
 		}
 	}
-	// Set basic authentication for the request if there is any.
-	if r.basicAuth != nil {
-		req.SetBasicAuth(r.basicAuth.username, r.basicAuth.password)
-	}
+	//// Set basic authentication for the request if there is any.
```

Comment:
Remove the commented code if it's not kept on purpose.

Reply:
This is a change in the turbo-api and should be handled there.